SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJobWebRunner/BrokerProfileOptOutSubJobWebRunner.swift

```diff
@@ -76,10 +82,17 @@ final class OptOutJob: DataBrokerJob {
         self.pixelHandler = pixelHandler
     }
 
-    func run(inputValue: ExtractedProfile,
-             webViewHandler: WebViewHandler? = nil,
-             actionsHandler: ActionsHandler? = nil,
-             showWebView: Bool = false) async throws {
+    public func optOut(profileQuery: BrokerProfileQueryData,
+                       extractedProfile: ExtractedProfile,
+                       showWebView: Bool,
+                       shouldRunNextStep: @escaping () -> Bool) async throws {
+        try await run(inputValue: extractedProfile, showWebView: showWebView)
+    }
+
+    public func run(inputValue: ExtractedProfile,
+                    webViewHandler: WebViewHandler? = nil,
+                    actionsHandler: ActionsHandler? = nil,
```

Comment:
Do you know when we ever pass through handlers for these?

Reply:
Terrible news: I removed the handlers locally, and the app compiles just fine. I think that means that these are used solely for the test suite, which means I want to find a way to remove them.

Reply:
Yeah, or at the very least clearly marginalize them (E.g. make the name explicit that that's all they're for)